CONTRIBUTING.md

```diff
@@ -280,17 +283,15 @@ sdk\<service name>\<package name>\*samples*
 e.g.
 
 ```
-sdk\eventgrid\Microsoft.Azure.EventGrid\src\Microsoft.Azure.EventGrid.csproj
-sdk\eventgrid\Microsoft.Azure.EventGrid\tests\Microsoft.Azure.EventGrid.Tests.csproj
-sdk\eventgrid\Microsoft.Azure.Management.EventGrid\src\Microsoft.Azure.Management.EventGrid.csproj
-sdk\eventgrid\Microsoft.Azure.Management.EventGrid\tests\Microsoft.Azure.Management.EventGrid.Tests.csproj
+sdk\eventgrid\Azure.Messaging.EventGrid\src\Azure.Messaging.EventGrid.csproj
+sdk\eventgrid\Azure.Messaging.EventGrid\tests\Azure.Messaging.EventGrid.Tests.csproj
+sdk\eventgrid\Azure.Management.EventGrid\src\Azure.Management.EventGrid.csproj
```

Comment:
Do these projects actually exist yet?

Reply:
It does not. Seems like the naming convention is different than the guidelines. What does exist is: `Azure.ResourceManager.AppConfiguration`. So, I'll change it to that.

Reply:
Fixed. Used actual projects that exist (AppConfiguration).